base/system/services/rpcserver.c

```diff
@@ -6725,7 +6740,7 @@ RI_ScValidatePnPService(
     hManager = ScmGetServiceManagerFromHandle(hSCManager);
     if (hManager == NULL)
     {
-        DPRINT1("Invalid handle!\n");
+        DPRINT("Invalid handle!\n");
```

Comment:
Last nitpick (1 of 2) before I will signal my approval:
```suggestion
        DPRINT("Invalid handle\n");
```

Reply:
Changed. Thanks.